modeldb/modeldb-run.yaml

```diff
@@ -1894,3 +1894,5 @@
   comment: ".mod files do not compile with NEURON 9.0"
 2016658:
   model_dir: 'MEC_model_figs5_6'
+2016662:
+  model_dir: 'code/radius = 6/20 fibers_same diameter_misaligned/dist = 0.1'
```

Comment:
I must not understand ```model_dir```.  That seems like a really strange folder name.

Reply:
It's a rather unfortunate one indeed! Apparently the CI passes, but looking at the raw report, it says:

```plaintext
"Exception: Explicitly specified model_dir %model_dir%/code/radius = 6/20 fibers_same diameter_misaligned/dist = 0.1 does not exist"
```

I think I'll need to give it a closer look.